docs/src/app/pages/MicroBitUart.md

```diff
@@ -1,10 +1,10 @@
-# Microbit_Uart
+# Microbit\_Uart
 
-Component for Microbit_Uart
+The <code>Microbit_Uart</code> sensor provides App Inventor users with the ability to read from and write strings to the BBC micro:bit's serial UART port.
```

Comment:
why mention users? Just say "provides" 
(Also, Hal came in today with a new mission that will be against the word "users")